datastore/redis.go

```diff
@@ -146,6 +154,50 @@ func (r *RedisCache) NumRegisteredValidators() (int64, error) {
 	return r.client.HLen(context.Background(), r.keyValidatorRegistrationTimestamp).Result()
 }
 
+func (r *RedisCache) SetActiveValidator(pubkeyHex types.PubkeyHex) error {
+	key := r.keyActiveValidators(time.Now())
+	err := r.client.HSet(context.Background(), key, PubkeyHexToLowerStr(pubkeyHex), "1").Err()
+	if err != nil {
+		return err
+	}
+
+	// set expiry
+	return r.client.Expire(context.Background(), key, expiryActiveValidators).Err()
+}
+
+func (r *RedisCache) NumActiveValidators() (uint64, error) {
+	hours := 2
```

Comment:
Hm, why is it 2 hours and not `expiryActiveValidators`? Also since it's used in multiple places I guess it should also be a constant

Reply:
done, using `expiryActiveValidators` now